pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -34,20 +34,21 @@ func (f *Factory) NewCmdClusterInfo(out io.Writer) *cobra.Command {
 		Short: "Display cluster info",
 		Long:  "Display addresses of the master and services with label kubernetes.io/cluster-service=true",
 		Run: func(cmd *cobra.Command, args []string) {
-			RunClusterInfo(f, out, cmd)
+			err := RunClusterInfo(f, out, cmd)
+			util.CheckErr(err)
```

Comment:
I personally would go through and change all of these to `util.CheckErr(RunClusterInfo(f, out, cmd))` - feels more idiomatic to me. Up to you though.